### DISCUSSION TITLE
rtl support

### DISCUSSION THREAD

**Discussion by thomaskous** [garronej/tss-react#134] [Q&A] [answered]
is there rtl support?

**Comment by garronej** [comment, marked as answer]
Hello @thomaskous,  

No, not yet but thank you for the question.  
I'll be interested in adding RTL support.  

Whay would be a fitting API in your oppigion?  
A provider would integrate nicely with i18n libraries, something like:   

```tsx
import { TssDirectionProvider } from "tss-react";
import {  useLang } from "i18nifty"; //For example

export function App(){
    const { lang } = useLang();
    return ( 
        <TssDirectionProvider direction={lang === "ar" ? "ltr" : "rtl"}> 
            {/* ... rest of the app */}
       </TssDirectionProvider>
    );
);
```

I could use this a reference:  https://github.com/alitaheri/jss-rtl    

Anyway, no it's not supported yet, PR are wellcome.
